src/lib/containers/IconSvg.tsx

```diff
@@ -102,12 +103,16 @@ const getIcon = (options:IconSvgOptions) => {
       return <AddShoppingCart style={muiSvgStyle}></AddShoppingCart>
     case 'addCircleOutline':
       return <AddCircleOutline style={muiSvgStyle}></AddCircleOutline>
+    case 'addCircleTwoTone':
+      return <AddCircleTwoTone style={muiSvgStyle}></AddCircleTwoTone>
```

Comment:
Used in a button in the new component:

![image](https://user-images.githubusercontent.com/17580037/142488941-fc3044e9-6b66-411b-85ad-25c5242ce30d.png)


The remaining changes are from code formatter